### ISSUE TITLE
Channels missmathing 

### ISSUE THREAD

**Issue by migleapa** [kwikteam/klusta#54]
Hello, 
I have this issue with kulsta, channels on clusterview doesn't match channels on waveformview. As you can see in the picture, I selected id 3 which is channel one, and on waveformview you can see it shows me channel 19. Can someone help me with that please?
Thanks,
Migle 
![klusta](https://user-images.githubusercontent.com/32569721/44782642-6ba16f80-ab77-11e8-9b68-74ca84169f96.png)

**Comment by rossant** [comment]
does this happen on the development version?

**Comment by migleapa** [comment]
Hello,
No, not the development one.

**Comment by migleapa** [comment]
Hello,

Just tried doing that, now it seems that channels are finally matching their waveforms but I have a new issue. If you look at the clusterview window it says that groups are unsorted? Not quite sure what that means? Does it mean that the clusters haven't been assigned to unique cluster groups? 

Thanks,

Migle
![klusta3](https://user-images.githubusercontent.com/32569721/45079290-aa887580-b0e1-11e8-80a9-a73a6b83c1a2.png)

**Comment by rossant** [comment]
Yes, by default clusters are not assigned a particular group. Phy allows you to specify groups for clusters: good, noise, MUA. (see the "wizard" menu)

**Comment by migleapa** [comment]
Hello, ok ok, it means everything is working! thank you!

**Comment by migleapa** [comment]
Hello. 
I have one more question. How do I interpret the featureview graphs. I have noticed they differ from non development version.
Thanks
